crates/cli/src/api.rs

```diff
@@ -97,6 +96,13 @@ pub struct StmtResultJson<'a> {
     pub stats: StmtStatsJson,
 }
 
+#[derive(Debug, Clone, Deserialize, Default)]
+pub struct StmtStatsJson {
+    pub rows_inserted: u64,
+    pub rows_deleted: u64,
+    pub rows_updated: u64,
+}
```

Comment:
I feel like API types are exactly the kind of thing that should be shared between the sender and receiver code. Do you know why this type is inside `core` instead of `client-api` in the first place? :thinking:

Reply:
No, I was planning to put it there as a follow-up.

Reply:
Done. Fwiw, there are plenty of other structs that are duplicated between cli and client-api, but those can definitely probably wait until a followup pr.

Reply:
ah rip okay